docs/references/api/deviceshifu-deployment.md

```diff
@@ -72,3 +72,18 @@ For a detailed example, please refer to https://github.com/Edgenesis/shifu/tree/
     - **IP_CAMERA_PASSWORD** (string)<br/>password for the camera RTSP stream.
     - **IP_CAMERA_CONTAINER_PORT**
         port of the RTSP camera driver container, which has to be `11111` (for now).
+
+## LwM2M-Gateway DeviceShifu Deployment
+
+For a detailed example, please refer to  [https://github.com/Edgenesis/shifu/tree/main/examples/lwm2m_gw_http](https://github.com/Edgenesis/shifu/tree/main/examples/lwm2m_gw_http).
+
+- **spec.template.spec.containers[0].image**
+  - **name** (string)<br/>the deviceShifu-HTTP container image enables devices using the HTTP protocol to connect to the LwM2M IoT management system. It performs protocol conversion and adaptation, allowing devices with different protocols to be uniformly managed and communicated through the LwM2M protocol. The gateway-driven test version of the driver image is `edgehub/deviceshifu-http-http:nightly`. Please use a stable version in production environments.
+- **spec.template.spec.containers[0].image.ports**
+  - **containerPort**<br/>`containerPort` specifies the port on which the application inside the container should listen. Such as `8080`.
+- **spec.template.spec.containers[1].image**
+  - **name** (string)<br/>the image for the LwM2M-gateway. The LwM2M-gateway serves as a protocol translation and data intermediary component that converts heterogeneous protocol device data into LwM2M protocol format, facilitating unified device management and control through the LwM2M server infrastructure. The test version of the driver image for the gateway driver is `edgehub/gateway-lwm2m:nightly`. Please use the stable version in the production environment.
```

Comment:
I won't explain what a gateway is here, just tell the user what will be populated.